app/portainer/views/endpoints/create/createEndpointController.js

```diff
@@ -54,10 +54,10 @@ angular
       const agentShortVersion = getAgentShortVersion(agentVersion);
 
       const deployCommands = {
-        kubeLoadBalancer: `curl -L https://downloads.portainer.io/portainer-agent-ce${agentShortVersion}-k8s-lb.yaml -o portainer-agent-k8s.yaml; kubectl apply -f portainer-agent-k8s.yaml`,
-        kubeNodePort: `curl -L https://downloads.portainer.io/portainer-agent-ce${agentShortVersion}-k8s-nodeport.yaml -o portainer-agent-k8s.yaml; kubectl apply -f portainer-agent-k8s.yaml`,
-        agentLinux: `curl -L https://downloads.portainer.io/agent-stack-ce${agentShortVersion}.yml -o agent-stack.yml && docker stack deploy --compose-file=agent-stack.yml portainer-agent`,
-        agentWindows: `curl -L https://downloads.portainer.io/agent-stack-ce${agentShortVersion}-windows.yml -o agent-stack-windows.yml && docker stack deploy --compose-file=agent-stack-windows.yml portainer-agent`,
+        kubeLoadBalancer: `curl -L https://downloads.portainer.io/ce${agentShortVersion}/portainer-agent-k8s-lb.yaml -o portainer-agent-k8s.yaml; kubectl apply -f portainer-agent-k8s.yaml`,
+        kubeNodePort: `curl -L https://downloads.portainer.io/ce${agentShortVersion}/portainer-agent-k8s-nodeport.yaml -o portainer-agent-k8s.yaml; kubectl apply -f portainer-agent-k8s.yaml`,
+        agentLinux: `curl -L https://downloads.portainer.io/ce${agentShortVersion}/agent-stack-ce.yml -o agent-stack.yml && docker stack deploy --compose-file=agent-stack.yml portainer-agent`,
```

Comment:
should this one be: `https://downloads.portainer.io/ce${agentShortVersion}/agent-stack.yml` instead?

Reply:
Agree, we don't need the agentShortVersion appended to the file name as its already in a folder specific for that version.

Reply:
Do we need agentShortVersion for `portainer-ce${agentVersion}-edge-agent-setup.sh` in app/portainer/views/endpoints/edit/endpointController.js ?